2/contrib/openshift/base-plugins.txt

```diff
@@ -15,13 +15,14 @@ jira:3.5
 job-dsl:1.77
 junit:1.52
 lockable-resources:2.11
+mailer:408.vd726a_1130320 
```

Comment:
I am curious if the core jenkins bump pulls in this plugin version anyway @adambkaplan , given this is a "special case core plugin"

Assuming this is the one of the 4 items you are referring to, I may take a stab pre-merge of removing this line and see if the correct version is pulled in anyway.

Generally, the less number of plugins we cite/pin in this file, the better.

Reply:
@adambkaplan and I talked in team meeting.  We'll save this experiment for a separate PR.